lib/ecl/ecl_sum_data.c

```diff
@@ -1205,25 +1203,40 @@ void ecl_sum_data_fwrite_interp_csv_line(const ecl_sum_data_type * data, time_t
   ecl_sum_data_init_interp_from_sim_time(data, sim_time, &time_index1, &time_index2, &weight1, &weight2);
 
   for (int i = 0; i < num_keywords; i++) {
-    double value = ecl_sum_data_vector_iget( data, sim_time, keylist , i , time_index1, time_index2, weight1, weight2);
+    int params_index = ecl_sum_vector_iget_param_index(keylist, i);
+    if (params_index >= 0) {
```

Comment:
I need it here?  We don't need to _expose_ the function out of this file, but I am confused as to whether `params_index=0` is valid or invalid.  If we had an "iget valid" function, that would be beyond doubt?

Reply:
>  If we had an "iget valid" function, that would be beyond doubt?

Sure - I will add it.